src/mca/bfrops/base/bfrop_base_frame.c

```diff
@@ -65,8 +65,6 @@ static int pmix_bfrop_register(pmix_mca_base_register_flag_t flags)
 {
     if (PMIX_MCA_BASE_REGISTER_DEFAULT == flags) {
         /* do something to silence warning */
```

Comment:
You'll need to change this or we'll get errors from other environments. Just change that little code block to:
```c
PMIX_HIDE_UNUSED_PARAMS(flag);
```